lib/logger.js

```diff
@@ -1,27 +1,46 @@
 'use strict';
 
 const Winston = require('winston');
+const morgan = require('morgan');
+
+function Logger(level, filename) {
+  const logLevel = level || 'info';
 
-function Logger(config) {
   const logger = new Winston.Logger({
-    level: config.get('log:level'),
+    level: logLevel,
     transports: [
       new Winston.transports.Console({
         colorize: true,
         timestamp: true
       })
-    ]
+    ],
+    exitOnError: false
   });
 
-  const filename = config.get('log:filename');
-
   if (filename) {
-    logger.add(Winston.transports.File, {
-      filename
-    });
+    logger.add(Winston.transports.File, {filename});
```

Comment:
Yes. Because logging through upstart doesn't require any extra log-rotation configurations, doesn't require any integration with the logging module for file-truncation, and really, Just Works :tm:.